_common.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxfo pipefail;
+
+USERNAME="${DOCKER_USERNAME:-jackfirth}"
+
+find_images () {
```

Comment:
This won't return the racket image tagged `latest` correct? Should it?

Reply:
Yes, it should!